tests/sender/testfile_lookup_with_quotes.csv

```diff
@@ -0,0 +1,5 @@
+KEY,DATA
+Key_one,Message_one
+Key_two,Message_two
+Key_three,Message_thre
+Key_four,Monitor 24" and 19"
```

Comment:
**nitpick:** please, add a newline at the end of the file.

Reply:
Yes, sir!